swagger2.0/oic.r.rule.expression.swagger.json

```diff
@@ -0,0 +1,153 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OCF Rule Expression",
+    "version": "20190910",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "copyright 2019 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/RuleExpressionResURI": {
+      "get": {
+        "description": "Expression for the Rule that defines the Rule logic in terms of the defined Rule Inputs, and which evaluates to a boolean value, for which \"true\" means that the Rule has been triggered.",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the Rule expression.",
+            "x-example":
+            {
+              "rt": ["oic.r.rule.expression"],
+              "if": ["oic.if.rw", "oic.if.baseline"],
+              "rule": "(timeperiod:triggered = true)",
```

Comment:
I'm unclear about timeperiod:triggered.

In the CR, the examples use "mytemperature:tempuratue".  This appears to be "rule-defined name":"OCF-defined Property Name".

In this example, it appears to be the opposite.

I'm also unclear how the Rule Input "timeperiod/triggered" could be tied to a specific Property in an OCF-defined Resource Type.  

Maybe someone needs to walk me through this.  I am putting similar comments in the CR itself.

Reply:
"timeperiod" lines up with the contents of an "anchor" in the collection of rule inputs.  That "anchor" gives you a link to a Resource, the Property is then that Property on that Resource.

Reply:
What might be confusing is the "triggered" is not a Property Name in the time period Resource Type.  Where did "triggered" come from?